src/actions/index.js

```diff
@@ -172,3 +172,16 @@ export function RSVP(RSVPRecord, history) {
             });
     };
 }
+
+// Add employment history
+export function addEmployment(personID,companyID,startDate,endDate, history) {
+    return (dispatch) => {
+        axios.post(`${ROOT_URL}/users/employment/add`, personID)
```

Comment:
Did you add the /users/employment/add endpoint to the backend? I didn't see a PR there